packages/main/src/plugin/extension-loader.ts

```diff
@@ -293,12 +295,12 @@ export class ExtensionLoader {
 
     const analyzedFoldersExtension = (
       await Promise.all(folders.map(folder => this.analyzeExtension(folder, false)))
-    ).filter(extension => extension !== undefined) as AnalyzedExtension[];
+    ).filter(extension => !extension.error) as AnalyzedExtension[];
     analyzedExtensions.push(...analyzedFoldersExtension);
 
     const analyzedExternalExtensions = (
       await Promise.all(externalExtensions.map(folder => this.analyzeExtension(folder, false)))
-    ).filter(extension => extension !== undefined) as AnalyzedExtension[];
+    ).filter(extension => !extension.error) as AnalyzedExtension[];
```

Comment:
```suggestion
    ).filter(extension => !extension.error);
```

Reply:
Good catch, fixed all three instances in latest commit.